app/assets/stylesheets/trace_view/_right_menu.less

```diff
@@ -21,6 +21,18 @@
   }
 }
 
+.info-tab-content {
+  padding: 10px;
+}
+
+.lm_header {
+  height: 20px !important;
```

Comment:
Please change this value in the `default_layout_config.js` instead. The `headerHeight` is defined there.